### PR TITLE
 `chore`: Integration test for Deleted Namespace

### PR DESCRIPTION
**Pull request by SiddeshUbale**
This PR contains an Integration test for Deleted Namespace.

This test follows the following flow:

- Get the JWT token and headers for Alpha1 NameSpace0
- Get the JWT token and headers for Alpha2 NameSpace0
- Create 4 Namespaces on Alpha1
- Add 10 items on Alpha1 Namespace1
- Add 10 items on Alpha1 Namespace2
- Add 10 items on Alpha1 Namespace3
- Add 10 items on Alpha1 Namespace4
- Delete the 2nd last namespace added on Alpha1
- Backup from Alpha1
- Restore on Alpha2
- Create 1 namespace on Alpha2
- Check if (newNamespaceID on Alpha2 > 2ndLastAddedNamespaceID on Alpha1)
- Check if data is preserved on namespaces.
- Check if 2ndLastAddedNamespaceID is not in Alpha2 namespaceId list.

### CODE REVIEW

**Comment by anurags92** [comment]
Please add more details on the test, the scenario it checks etc.

File: systest/backup/advanced-scenarios/deleted-namespace/backup_test.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	jwtTokenAlpha1Np2, headerAlpha1Np2 := utilsCommon.GetJwtTokenAndHeader(t, "alpha1", 2)
+	utilsCommon.AddItemSchema(t, headerAlpha1Np2, "alpha1")
+	utilsCommon.AddItem(t, 21, 30, jwtTokenAlpha1Np2, "alpha1")
+	jwtTokenAlpha1Np3, headerAlpha1Np3 := utilsCommon.GetJwtTokenAndHeader(t, "alpha1", 3)
+	utilsCommon.AddItemSchema(t, headerAlpha1Np3, "alpha1")
+	utilsCommon.AddItem(t, 31, 40, jwtTokenAlpha1Np3, "alpha1")
+	jwtTokenAlpha1Np4, headerAlpha1Np4 := utilsCommon.GetJwtTokenAndHeader(t, "alpha1", 4)
+	utilsCommon.AddItemSchema(t, headerAlpha1Np4, "alpha1")
+	utilsCommon.AddItem(t, 41, 50, jwtTokenAlpha1Np4, "alpha1")
+	e2eCommon.DeleteNamespace(t, ns2[1], headerAlpha1Np0, "alpha1")
```
Comment (anurags92) [inline systest/backup/advanced-scenarios/deleted-namespace/backup_test.go:36]: Delete namespace `ns2[0]` instead of the last namespace and check that the new namespace is still greater than `ns2[1]`.

Comment (SiddeshUbale) [reply systest/backup/advanced-scenarios/deleted-namespace/backup_test.go]: Resolved

File: systest/backup/advanced-scenarios/deleted-namespace/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    - 6080
+    volumes:
+    - type: bind
+      source: $GOPATH/bin
+      target: /gobin
+      read_only: true
+    - data-volume:/data/backups/
+    command: /gobin/dgraph  ${COVERAGE_OUTPUT} zero --raft "idx=1;" --my=zero2:5080 --logtostderr -v=2 --bindall
+volumes:
+  data-volume:
```
Comment (mangalaman93) [inline systest/backup/advanced-scenarios/deleted-namespace/docker-compose.yml:85]: add a newline at the end of the file

File: systest/backup/common/utils.go [hunk 3]
```diff
@@ -104,14 +104,14 @@ func CopyToLocalFs(t *testing.T) {
...
 		Query: `mutation {
-			    updateGQLSchema(
-			      input: { set: { schema: "type Item {id: ID!, name: String! @search(by: [hash]), price: String!}"}})
-			    {
-			      gqlSchema {
-					schema
-			      }
-			    }
-			  }`,
+			updateGQLSchema(
```
Comment (mangalaman93) [inline systest/backup/common/utils.go:107]: undo this change

**Review by mangalaman93 — CHANGES_REQUESTED** [review]
minor comments. Looks good otherwise

**Review by anurags92 — APPROVED** [review]

**Review by mangalaman93 — APPROVED** [review]
